notes/interview.vbs

```diff
@@ -9329,6 +9391,55 @@ Loop until are_we_passworded_out = FALSE
 
 Call check_for_MAXIS(False)
 
+If need_to_update_addr = "True" then need_to_update_addr = True
+If need_to_update_addr = "False" then need_to_update_addr = False
+
+If need_to_update_addr = "True" Then 
+    Call access_ADDR_panel("WRITE", notes_on_address, resi_line_one, resi_line_two, resi_addr_street_full, resi_addr_city, resi_addr_state, resi_addr_zip, resi_addr_county, addr_verif, homeless_yn, reservation_yn, living_situation, reservation_name, mail_line_one, mail_line_two, mail_addr_street_full, mail_addr_city, mail_addr_state, mail_addr_zip, address_change_date, addr_future_date, phone_one_number, phone_two_number, phone_three_number, phone_one_type, phone_two_type, phone_three_type, text_yn_one, text_yn_two, text_yn_three, addr_email, verif_received, original_information, update_attempted)
+End If
+
+'Units can have either the heat/air standard utility deduction OR one or both of the electric and phone standard utility deduction(s) when they are not responsible for heating and/or cooling expenses.
+choice_date = CAF_datestamp
+hest_update_heat_ac 	= FALSE 
+hest_update_electric 	= FALSE 
+hest_update_phone 		= FALSE
+
+If question_15_heat_ac_yn = "Yes" Then 	'If heat/ac is paid for then set phone and electric to blank because we can only claim heat/ac
+	hest_update_heat_ac = TRUE
+	prosp_heat_ac_yn = "Y"
+	prosp_electric_yn = " " 
+	prosp_electric_units = "  "
+	prosp_phone_yn = " "
+	prosp_phone_units = "  "
+
+Else 									'If heat/ac is not paid for, then electric and/or phone can be claimed 
+	prosp_heat_ac_yn = " "
+	prosp_heat_ac_units = "  "
+
+	If question_15_electricity_yn = "Yes" Then 
+		hest_update_electric = TRUE
+		prosp_electric_yn = "Y" 
+	
+	Else
+		prosp_electric_yn = " " 
+		prosp_electric_units = "  "
+	End If 
+
+	If question_15_phone_yn = "Yes" Then 
+		hest_update_phone = TRUE
+		prosp_phone_yn = "Y"
+	Else
+		prosp_phone_yn = " "
+		prosp_phone_units = "  "
```

Comment:
Looks like the function doesn't blank out the number of units, which errors the panel. What does work is saying "N" and leaving the "01" on the panel when updating - so changing these to
 Else
Prosp_phone_yn = "N"
should actually be the working solution